playpen/ansible/roles/dev/tasks/main.yml

```diff
@@ -55,3 +55,5 @@
 
 - name: Install custom ~/.bashrc
   copy: src=bashrc dest=/home/{{ ansible_env.SUDO_USER }}/.bashrc
+
```

Comment:
(submitted before I was done, sorry)

...but it lines up with the rest of the formatting, so it's for the best.